dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/search/SearchFields.java

```diff
@@ -261,4 +261,6 @@ public class SearchFields {
     public static final String LICENSE = "license";
     public static final String EMBARGO_UNTIL = "embargoUntil";
 
+    public static final String GEOGRAPHIC_COORDINATES = "dsf_txt_geographicCoordinates";
```

Comment:
Maybe we can use `SearchDynamicFieldPrefix`?
GEOGRAPHIC_COORDINATES = SearchDynamicFieldPrefix.TEXT.getPrefix() +  "geographicCoordinates";